src/main/java/com/ldtteam/blockout/Loader.java

```diff
@@ -37,7 +37,7 @@
         register("text", Text.class);
         register("button", ButtonVanilla.class);
         register("buttonimage", ButtonImage.class);
-        register("label", Label.class);
+        register("label", Text.class); // we don't want to deal with xml changes
```

Comment:
maybe add a "todo" remove in the future or so

Reply:
will probably nuke it too, cuz with all these three PRs and upcoming Shadow's changes it's gonna be a quite big breaking change already